src/common/createSlider.jsx

```diff
@@ -215,7 +215,7 @@ export default function createSlider(Component) {
     calcOffset(value) {
       const { min, max } = this.props;
       const ratio = (value - min) / (max - min);
-      return ratio * 100;
+      return isNaN(ratio) ? 0 : ratio * 100;
```

Comment:
I don't konw in what case ratio could be `NaN`

- `max - min` should always more than `0` if uses give legal props

- `value - min` should always no less than `0`

So i think ratio should be always no less than `0` and no more than `1`